origami.json

```diff
@@ -18,6 +18,7 @@
 	"keywords": "interstitial messaging modal popover notification hover",
 	"browserFeatures": {
 		"required": [
+			"array.from",
```

Comment:
We can't require a new feature without releasing a major, as our users will have to take action to include the poyfill -- sorry should have thought to have mentioned earlier, I assumed you we working on o-comments 😅 

As you're away I'll refactor to remove `array.from` tomorrow 👍

Reply:
Thanks @notlee I wasn't aware of that. Thanks for fixing